toolkit/offchain/tests/integration_tests.rs

```diff
@@ -273,24 +274,28 @@ async fn run_upsert_d_param<
 	num_permissioned_candidates: u16,
 	num_registered_candidates: u16,
 	pkey: &CardanoPaymentSigningKey,
+	governance: Vec<MainchainKeyHash>,
 	client: &T,
 ) -> Option<McTxHash> {
 	let tx_hash = d_param::upsert_d_param(
 		genesis_utxo,
 		&DParameter { num_permissioned_candidates, num_registered_candidates },
 		pkey,
+		governance,
 		client,
 		&FixedDelayRetries::new(Duration::from_millis(500), 100),
 	)
 	.await
 	.unwrap();
-	if let Some(tx_hash) = tx_hash {
+	if let Some(TxHash(tx_hash)) = tx_hash {
```

Comment:
Cosmetic: `TxHash(tx_hash)` is confusing, could you use qualified name for the outer one, please? `McSmartContractResult::TxHash`